Core/clim-basic/geometry/region-bounding-rectangles.lisp

```diff
@@ -46,7 +46,7 @@
                                    (expt (* h cos) 2))))))))
       (values (- cx x) (- cy y) (+ cx x) (+ cy y)))))
 
-(defmethod bounding-rectangle* ((region standard-ellipse))
+(defmethod bounding-rectangle* ((region elliptical-thing))
```

Comment:
At the end of `Tests/regions.lisp`, there are tests for elliptical regions. Would it make sense to test this method there?

Reply:
willdo

Reply:
I've added tests to Tests/bounding-rectangles.lisp (done)